civil/civil.go

```diff
@@ -107,6 +107,11 @@ func (d1 Date) After(d2 Date) bool {
 	return d2.Before(d1)
 }
 
+// IsEmpty reports whether date fields are empty
+func (d Date) IsEmpty() bool {
```

Comment:
I think we should match the API of the time package and use [IsZero](https://pkg.go.dev/time#Time.IsZero).

Reply:
done